src/hooks/useLoadInitialData.tsx

```diff
@@ -42,7 +42,7 @@ export const useLoadInitialData = () => {
     useEffect(() => {
         const unsubscribe = serverSubscribeUpdates(clientErrorHandler);
         return unsubscribe;
-    }, [serverSubscribeUpdates, documentDrives.length]);
+    }, [serverSubscribeUpdates, documentDrives]);
```

Comment:
will this not cause the hook to not run if something other than the number of drives changes? Asking because I am not so familiar with this code.

Reply:
The document drives array is refreshed when the state of the drives change, or a drive is removed/added.
https://github.com/powerhouse-inc/connect/blob/34f457234578fe152a0f26259b214ad0c959132e/src/hooks/useDocumentDrives.ts#L35

I guess the issue was that a drive was replaced but since the array length was the same, this was not subscribing to the new drive.

Reply:
> ss the issue was that a drive was replaced but since the array length was the same, this was not subscribing to the new drive.

that's exactly what was happening here